cmd/influx/task.go

```diff
@@ -207,15 +210,20 @@ func taskFindF(cmd *cobra.Command, args []string) error {
 var taskUpdateFlags struct {
 	id     string
 	status string
+	file   string
 }
 
 func taskUpdateCmd(opt genericCLIOpts) *cobra.Command {
 	cmd := opt.newCmd("update", taskUpdateF, true)
 	cmd.Short = "Update task"
+	cmd.Long = `Update task
+An optional Flux script can be provided as an argument or 
+in a file using the -f flag, '-' argument reads the Flux script from stdin.`
```

Comment:
Update task status or script. Provide a Flux script via stdin or a file.

Is the "`-` from stdin" still true?

Reply:
Yes, both '-' and '@file' are still supported for backward compatibility reasons. Herein, If the user wants to update the script from std-in, "-" must be an argument to let the tool know that the flux script comes from stdin.